Vienna/Sources/Application/ViennaApp.m

```diff
@@ -36,6 +36,8 @@
 @interface ViennaApp ()
 
 @property (nonatomic) VNAFeedListSizeMode feedListSizeMode;
+@property (nonatomic, copy) NSString *articleListFontName;
+@property (nonatomic) NSInteger articleListFontSize;
 @property (readonly, nonatomic) BOOL readingPaneOnRight;
```

Comment:
I am not 100% comfortable with having these properties declared in ViennaApp.m instead of ViennaApp.h

It hides the fact that these are needed in Vienna.sdef for AppleScript support